src/app/pages/swap/swap.context.ts

```diff
@@ -2,11 +2,19 @@ import { createContext, useContext } from 'react';
 
 import { SwapAsset, SwapFormValues } from './hooks/use-swap';
 
+export interface SwapSubmissionData {
+  swapAmountFrom: string;
+  swapAmountTo: string;
+  swapAssetFrom: SwapAsset;
+  swapAssetTo: SwapAsset;
+  liquidityFee: number;
+  router: SwapAsset[];
+  slippage: number;
+}
+
 export interface SwapContext {
-  exchangeRate: number;
-  isSendingMax: boolean;
-  onSetExchangeRate(value: number): void;
-  onSetIsSendingMax(value: boolean): void;
```

Comment:
Working on this locally, curious why you removed all the code to set the send max? It seemed like a good way to toggle that imo and utilize the state throughout the feature? I think we want to keep it bc others will be using this code too.